File: src/question/month08/day0812/ms_0812.cpp
#include <iostream>

using namespace std;

int main() {
	int data[10], odd = 0, even = 0;

	for (int i = 0; i < sizeof(data) / sizeof(data[0]); i++) 
	{
		scanf_s("%d", &data[i]);

		if (i % 2 == 0)
			odd += data[i];
		else
			even += data[i];
	}

	cout << "odd : " << odd << endl;
	cout << "even : " << even << endl;

	return 0;
}

File: year2020/month08/day0812/ms_0812.cs
using System;

namespace algorithm_33
{
    class ms_0812
    {
        static void Main(string[] args)
        {
            int[] data = new int[10];
            int odd = 0, even = 0;

            for(int i = 0; i < data.Length; i++) 
            {
                data[i] = Convert.ToInt32(Console.ReadLine());

                if(i%2 == 0)
                    odd += data[i];
                else
                    even += data[i];
            }

            Console.WriteLine($"odd : {odd}\neven : {even}");
        }
    }
}
